### PR TITLE
Improvement/steppers

### PR DESCRIPTION
**Pull request by StheffanyHadlich**
**Component**: Steppers

**Description**:

Improvement change prop `title` 's type from `String` to `Node`, add `error` and `inProgress` prop

| inProgress | !inProgress|
|--------------- | ---------------|
| image | image |

**Design**:

**Breaking Changes**:

- [] Breaking Changes

---

See: #43

### CODE REVIEW

File: src/lib/components/steppers/Steppers.component.js [hunk 1]
```diff
@@ -43,6 +47,11 @@ const Circle = styled.div`
         background-color: ${brandingTheme.primary};
         color: ${defaultTheme.white};
       `;
+    } else if (props.error) {
+      return css`
+        background-color: red;
+        color: ${defaultTheme.white};
```
Comment (carlito-scality) [inline src/lib/components/steppers/Steppers.component.js]: `${props => mergeTheme(props.theme, defaultTheme).text};` to support dark mode

File: src/lib/components/steppers/Steppers.component.js [hunk 2]
```diff
@@ -43,6 +47,11 @@ const Circle = styled.div`
         background-color: ${brandingTheme.primary};
         color: ${defaultTheme.white};
       `;
+    } else if (props.error) {
+      return css`
+        background-color: red;
```
Comment (carlito-scality) [inline src/lib/components/steppers/Steppers.component.js]: `${props => mergeTheme(props.theme, defaultTheme).danger};` to be themeable

File: src/lib/components/steppers/Steppers.component.js [hunk 3]
```diff
@@ -43,6 +47,11 @@ const Circle = styled.div`
         background-color: ${brandingTheme.primary};
         color: ${defaultTheme.white};
       `;
+    } else if (props.error) {
```
Comment (carlito-scality) [inline src/lib/components/steppers/Steppers.component.js]: A step could be:
active => primary color
active and inProgress => primary color with loader
active and error => danger color
completed => green with check icon
default => `gray` color (next steps)

**Review by carlito-scality — CHANGES_REQUESTED** [review]

File: src/lib/components/steppers/Steppers.component.js (hunk 1 above)
Comment (StheffanyHadlich) [reply src/lib/components/steppers/Steppers.component.js]: @carlito-scality but if the background for error is always red, white wouldn't be a better color here?

**Review by carlito-scality — APPROVED** [review]
LGTM. merged in https://github.com/scality/core-ui/pull/148.
Thanks :)

**Pull request merged**